app/src/main/java/ch/epfl/sdp/peakar/user/services/AuthAccount.java

```diff
@@ -6,8 +6,8 @@
 
 import ch.epfl.sdp.peakar.points.CountryHighPoint;
 import ch.epfl.sdp.peakar.points.POIPoint;
+import ch.epfl.sdp.peakar.social.SocialItem;
 import ch.epfl.sdp.peakar.user.challenge.Challenge;
-import ch.epfl.sdp.peakar.user.friends.FriendItem;
 import ch.epfl.sdp.peakar.user.outcome.ProfileOutcome;
 
```

Comment:
Method `isEnrolledInAChallenge()` is never used

Reply:
As before, Bastien is working on the challenges part so I don't think we should remove anything related to it for now.